.docs/03-adding-structure.md

```diff
@@ -0,0 +1,11 @@
+[← Getting Started][back] - [:book:][index] - [Testing →][next]
+
+# Adding Structure
```

Comment:
Perhaps call this "JSON Encoding" concretely for discoverability, and write it as an introduction to structured encoding in abstract as well.

- Example
- Server
- Curl
- Yab
- Client